assisted-events-scrape/workers/__init__.py

```diff
@@ -0,0 +1 @@
+from .cluster_events_worker import ClusterEventsWorker, ClusterEventsWorkerConfig
```

Comment:
Need to declare `__all__` for defining what symbols in a module will be exported